packages/components/psammead-media-player/src/Placeholder/__snapshots__/index.test.jsx.snap

```diff
@@ -128,6 +128,222 @@ exports[`Media Player: Placeholder should render a video placeholder 1`] = `
 </div>
 `;
 
+exports[`Media Player: Placeholder should render a video placeholder with guidance 1`] = `
+.c14 {
+  display: block;
+  width: 100%;
+  visibility: visible;
+}
+
+.c12 {
+  vertical-align: middle;
+  margin: 0 0.25rem;
+  color: #222222;
+  fill: currentColor;
+  width: 0.75rem;
+  height: 0.75rem;
+}
+
+.c10 {
+  -webkit-clip-path: inset(100%);
+  clip-path: inset(100%);
+  -webkit-clip: rect(1px,1px,1px,1px);
+  clip: rect(1px,1px,1px,1px);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  width: 1px;
+  margin: 0;
+}
+
+.c8 {
+  background-color: #222222;
+  border: none;
+  color: #FFFFFF;
+  cursor: pointer;
+  display: block;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  height: 5rem;
+  padding: 0;
+  -webkit-transition: background-color 300ms;
+  transition: background-color 300ms;
+  width: 5rem;
+}
+
+.c8:hover,
+.c8:focus {
+  background-color: #B80000;
+  -webkit-transition: background-color 300ms;
+  transition: background-color 300ms;
+}
+
+.c11 > svg {
+  color: #FFFFFF;
+  fill: currentColor;
+  height: 1.5rem;
+  margin-top: 0.5rem;
+  width: 1.5rem;
+}
+
+.c13 {
+  display: block;
+  margin-top: 0.5rem;
+}
+
+.c6 {
+  color: #222222;
+  fill: currentColor;
+}
+
+.c2 {
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 700;
```

Comment:
The UX specification says that the guidance message should have a normal font weight. This suggests that the guidance message will have a weight of `700` (and it does in Storybook).